website/docs/docs/core/connect-data-platform/clickhouse-setup.md

```diff
@@ -22,87 +22,73 @@ import SetUpPages from '/snippets/_setup-pages-intro.md';
 
 <SetUpPages meta={frontMatter.meta} />
 
+
+## Supported features
+
+- [x] Table materialization
+- [x] View materialization
+- [x] Incremental materialization
+- [x] Microbatch incremental materialization
+- [x] Materialized View materializations (uses the `TO` form of MATERIALIZED VIEW, experimental)
+- [x] Seeds
+- [x] Sources
+- [x] Docs generate
+- [x] Tests
+- [x] Snapshots
+- [x] Most dbt-utils macros (now included in dbt-core)
+- [x] Ephemeral materialization
```

Comment:
Could we bring all the materialization types together? As a table that looks more like [this](https://docs.getdbt.com/docs/core/connect-data-platform/materialize-setup#supported-features)? We might not even need it, since we don't really deviate from the base materializations in any way (maybe save for materialized views).